docs/src/api/class-browsercontext.md

```diff
@@ -1287,7 +1287,7 @@ A glob pattern, regex pattern or predicate receiving [URL] used to register a ro
 ### param: BrowserContext.unroute.handler
 * since: v1.8
 * langs: js, python
-- `handler` ?<[function]\([Route], [Request]\)>
+- `handler` ?<[function]\([Route], [Request]\): [Promise<any>|any]>
```

Comment:
Why is return type not just `any` ?

Reply:
See https://github.com/microsoft/playwright/pull/20005